### PR TITLE
Cluster node selection should be properly random

### PR DESCRIPTION
**Pull request by qrilka**
Random generator used for node selection should be updated for subsequent `inCluster` operations

### CODE REVIEW

File: riak.cabal [hunk 1]
```diff
@@ -122,6 +123,7 @@ library
                 random-shuffle                >= 0.0.4 && < 0.1,
                 riak-protobuf                 == 0.21.*,
                 semigroups                    >= 0.16,
+                stm,
                 text                          == 1.2.*,
```
Comment (tmcgilchrist) [inline riak.cabal]: Could I get you to put some version ranges on `stm` and `transfromers-base` please?

Comment (qrilka) [inline riak.cabal]: Done

**Comment by k-bx** [comment]
@tmcgilchrist would appreciate if you check this out again. Thank you!

**Comment by tmcgilchrist** [comment]
Thanks @qrilka

**Comment by qrilka** [comment]
@tmcgilchrist any plans for a new release with this fix?

**Comment by tmcgilchrist** [comment]
Sure I'll bundle up a point release this morning.

**Pull request merged**